src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NpgService.kt

```diff
@@ -40,7 +40,9 @@ data class NgpOrderPendingStatus(
 @Service
 class NpgService(
   private val authorizationStateRetrieverService: AuthorizationStateRetrieverService,
-  @Value("\${npg.refund.delayFromAuthRequestMinutes}") val refundDelayFromAuthRequestMinutes: Long
+  @Value("\${npg.refund.delayFromAuthRequestMinutes}") val refundDelayFromAuthRequestMinutes: Long,
+  @Value("\${npg.refund.refundDelayForRefundOperationsSeconds}")
```

Comment:
```suggestion
  @Value("\${npg.refund.eventProcessingDelaySeconds}")
```
🚲  what about this parameter name?

Reply:
fixed in e2539bc77eed9dc5bf71673c2c5e15cc83854000